### PR TITLE
[TEVA-2165] Update documentation with extra information

### PR DESCRIPTION
**Pull request by felixtheflex**
## Jira ticket URL

https://dfedigital.atlassian.net/browse/TEVA-2165

## Changes in this PR:

Updated the documentation with extra information

- [ ] New development configuration to be shared?

### CODE REVIEW

**Comment by martin-bangoura** [comment]
Review app deployed to <https://teaching-vacancies-review-pr-3128.london.cloudapps.digital>

**Review by cpjmcquillan — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -48,7 +48,11 @@ asdf install
 Make sure you have the following services configured and running on your development background:
 
 * [PostgreSQL](https://www.postgresql.org)
-* [Redis](https://redis.io)
+* [Redis]*(https://redis.io)
```
Comment (saliceti) [inline README.md:51]: Remove extra *

File: README.md [hunk 2]
```diff
@@ -48,7 +48,11 @@ asdf install
...
+
+### *Redis
```
Comment (saliceti) [inline README.md:53]: Remove extra *

File: README.md [hunk 3]
```diff
@@ -48,7 +48,11 @@ asdf install
...
+You can check if Redis has been installed by - `redis-cli -v`
```
Comment (saliceti) [inline README.md:54]: This only checks the client is installed. We should start the server and test the connection. On mac:
```
% redis-server
5227:C 24 Mar 2021 11:01:45.921 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
...
% redis-cli                                                                                                                                                                                                                         11:02:11
127.0.0.1:6379> 
```

File: README.md [hunk 4]
```diff
@@ -50,6 +50,11 @@ Make sure you have the following services configured and running on your develop
 * [PostgreSQL](https://www.postgresql.org)
 * [Redis](https://redis.io)
 
+### Redis
+You can check if Redis client is installed by - `redis-server`
```
Comment (nickromneydfe) [inline README.md:54]: I think this is a typo - can check if Redis server is installed

**Comment by felixtheflex** [comment]
Closing - it was deemed perhaps the extra info doesnt add value.

**Pull request closed without merging**